### PR TITLE
RT 1.7 local_bgp_test.go - OTG file

### PR DESCRIPTION
**Pull request by stthummuri**
Porting changes from https://github.com/openconfig/featureprofiles/pull/730 to OTG location for same script. 

Added BGP Peer group config in both scripts.
Added network instance type to default in both scripts.

### CODE REVIEW

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 1]
```diff
@@ -71,38 +97,49 @@ func TestEstablish(t *testing.T) {
 	ate := ondatra.DUT(t, "dut2")
 	atePortName := ate.Port(t, "port1").Name()
 	intf2 := ateAttrs.NewInterface(atePortName)
-	ate.Config().Interface(intf2.GetName()).Replace(t, intf2)
+	gnmi.Replace(t, ate, gnmi.OC().Interface(intf2.GetName()).Config(), intf2)A
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:100]: Looks like there's a stray "A" at the end of this line

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 2]
```diff
@@ -132,27 +169,33 @@ func TestDisconnect(t *testing.T) {
...
+	gnmi.Replace(t, dut, dutConfPath.Config(), dutConf)
+	gnmi.Replace(t, ate, ateConfPath.Config(), ateConf)
+	time.Sleep(time.Second * 15)
+	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*30, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
 	// ateConfPath.Delete(t)
-	ateConfPath.Neighbor(dutIP).Enabled().Replace(t, false)
-	nbrPath.SessionState().Await(t, time.Second*15, telemetry.Bgp_Neighbor_SessionState_ACTIVE)
-	code := nbrPath.Messages().Received().LastNotificationErrorCode().Get(t)
+	gnmi.Replace(t, ate, ateConfPath.Neighbor(dutIP).Enabled().Config(), false)
+	time.Sleep(time.Second * 5)
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:196]: Same question about sleep/await as above

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 3]
```diff
@@ -132,27 +169,33 @@ func TestDisconnect(t *testing.T) {
...
 		PeerAs:          ygot.Uint32(dutAS),
 		NeighborAddress: ygot.String(dutIP),
+		PeerGroup:       ygot.String(peerGrpName),
 	})
-	dutConfPath.Replace(t, dutConf)
-	ateConfPath.Replace(t, ateConf)
-	nbrPath.SessionState().Await(t, time.Second*15, telemetry.Bgp_Neighbor_SessionState_ESTABLISHED)
+	gnmi.Replace(t, dut, dutConfPath.Config(), dutConf)
+	gnmi.Replace(t, ate, ateConfPath.Config(), ateConf)
+	time.Sleep(time.Second * 15)
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:192]: Why sleep for 5 seconds before awaiting? Can you just add 5 seconds to the await call?

File: feature/bgp/tests/local_bgp_test/local_bgp_test.go [hunk 4]
```diff
@@ -301,16 +370,21 @@ func TestParameters(t *testing.T) {
 			dutConfPath.Delete(t)
 			ateConfPath.Delete(t)
 			// Renable and wait to establish
-			dutConfPath.Replace(t, tc.dutConf)
-			ateConfPath.Replace(t, tc.ateConf)
-			nbrPath.SessionState().Await(t, time.Second*30, telemetry.Bgp_Neighbor_SessionState_ESTABLISHED)
+			gnmi.Replace(t, dut, dutConfPath.Config(), tc.dutConf)
+			gnmi.Replace(t, ate, ateConfPath.Config(), tc.ateConf)
+			time.Sleep(time.Second * 15)
+			gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*30, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
```
Comment (greg-dennis) [inline feature/bgp/tests/local_bgp_test/local_bgp_test.go:376]: same sleep/await question

**Review by greg-dennis — CHANGES_REQUESTED** [review]

**Comment by stthummuri** [comment]
Closing this since #875 has been opened for the same

**Comment by stthummuri** [comment]
Closing this since https://github.com/openconfig/featureprofiles/pull/875 has been opened for the same

**Pull request closed without merging**